Sources/Email/VaporEmail.swift

```diff
@@ -0,0 +1,70 @@
+import Vapor
+
+public struct VaporEmail: Content {
+    public let from: EmailAddress
+    public let to: [EmailAddress]
+    public let replyTo: EmailAddress?
+    public let cc: [EmailAddress]?
+    public let bcc: [EmailAddress]?
+    public let subject: String
+    public let text: String
+    public let html: String?
```

Comment:
Maybe an enum would be better here?
```swift
enum EmailMessage.Content {
    case text(String)
    case html(String)
}
```
Or maybe email should have headers / a content type?